src/main/java/org/zone/region/flag/entity/player/move/message/display/MessageDisplayTypes.java

```diff
@@ -0,0 +1,39 @@
+package org.zone.region.flag.entity.player.move.message.display;
+
+import org.jetbrains.annotations.NotNull;
+import org.zone.region.flag.entity.player.move.message.display.bossbar.BossBarMessageDisplayType;
+import org.zone.region.flag.entity.player.move.message.display.chat.ChatMessageDisplayType;
+import org.zone.region.flag.entity.player.move.message.display.title.TitleMessageDisplayType;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public final class MessageDisplayTypes {
+
+    public static final ChatMessageDisplayType CHAT_MESSAGE_DISPLAY = new ChatMessageDisplayType();
```

Comment:
The suffix of ``MESSAGE_DISPLAY`` can be removed as the class is called ``MessageDisplayTypes`` so the fact these are Message Display Types can be safely assumed